tests/end-to-end/director/testctx.go

```diff
@@ -24,33 +23,17 @@ func init() {
 	}
 }
 
-type scopeProvider interface {
-	GetRequiredScopes(path string) ([]string, error)
-	GetAllScopes() ([]string, error)
-}
-
 // testContext contains dependencies that help executing tests
 type testContext struct {
 	graphqlizer       graphqlizer
 	gqlFieldsProvider gqlFieldsProvider
-	scopeProvider     scopeProvider
 	currentScopes     []string
 	cli               *gcli.Client
 }
 
 func newTestContext() (*testContext, error) {
-	scopesCfgPath := os.Getenv("SCOPES_CONFIGURATION_FILE")
-	scopeProvider := scope.NewProvider(scopesCfgPath)
-
-	err := scopeProvider.Load()
-	if err != nil {
-		return nil, errors.Wrap(err, "while loading config for scopes")
-	}
-
-	currentScopes, err := scopeProvider.GetAllScopes()
-	if err != nil {
-		return nil, err
-	}
+	scopes := os.Getenv("SCOPES_CONFIGURATION")
```

Comment:
1. maybe rename it to ALL_POSSIBLE_SCOPES
2. document this env variable in README.md
3. inject this variable in the chart

Reply:
`ALL_SCOPES` is good name.